### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by pyhys**

### CODE REVIEW

**Review by olbjo — APPROVED** [review]

File: doc/getting_started_linux.rst [hunk 1]
```diff
@@ -160,14 +163,78 @@ To change the ephemeral port range::
...
+
+    p-net/build$ size pn_dev
+       text	   data	    bss	    dec	    hex	filename
+    337054	   1240	     24	 338318	  5298e	pn_dev
+
+See https://linux.die.net/man/1/size for information on how to use the command.
+Also the rt-kernel page in this documentation has some description on how to
+interpret the output.
+
+Add the ``-A`` flag to have more detailed information. If you build with debug
```
Comment (hefloryd) [inline doc/getting_started_linux.rst:185]: I don't see that size -A adds anything interesting?

File: doc/implementation_details.rst [hunk 2]
```diff
@@ -510,3 +517,8 @@ Include headers in sorted groups in this order:
 Avoid "Yoda conditions"::
 
     if (3 == a) { /* ... */ }
+
+Use C-style comments instead of C++ comments::
```
[the remark this answers is not in the document]
Comment (pyhys) [reply doc/implementation_details.rst]: Done

File: doc/getting_started_linux.rst (hunk 1 above)
Comment (pyhys) [reply doc/getting_started_linux.rst]: Removed

File: doc/getting_started_linux.rst [hunk 3]
```diff
@@ -160,14 +163,78 @@ To change the ephemeral port range::
...
+The resulting file size of the sample application binary is heavily dependent
+on the compile time options, for example whether to include debug information::
+
+    p-net/build$ ls -l pn_dev
+    -rwxr-xr-x 1 jonas jonas 365984 sep  9 15:37 pn_dev
+
+    p-net/build$ file pn_dev
+    pn_dev: ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, BuildID[sha1]=d2c45d5a17c1ad63db4f28c82cc04f8cfe5dcbfd, not stripped
+
+    p-net/build$ size pn_dev
```
[the remark this answers is not in the document]
Comment (pyhys) [reply doc/getting_started_linux.rst]: Updated, please read again.

**Pull request merged**